docs/angular/guides/setup-mfe-with-angular.md

```diff
@@ -0,0 +1,393 @@
+# How to setup a Micro Frontend with Angular and Nx
+
+[Webpack 5]() introduced a [Module Federation Plugin]() enabling multiple, independently built and deployed bundles of code to form a single application. This is the foundation of Micro Frontend Architecture and the Module Federation Plugin makes implementing such an architecture much simpler.  
+With Angular 12 adding support for Webpack 5 it increases the viability of scaffolding a Micro Frontend architecture with Angular.
+
+There are some challenges that exist, such as being able to be able to pass a custom Webpack configuration file, which is required to add the Module Federation configuration, to Angular's compiler.
+
+We recently made changes to our `@nrwl/angular:webpack-browser` executor that allows you to supply a path to a custom Webpack configuration file which we then merge with Angular's default Webpack configuration before telling Angular to use this merged configuration to build the Angular application.
+
+We also made some changes to our `@nrwl/angular:app` generator to aid in the scaffolding of Module Federation configuration required for setting up a Micro Frontend Architecture.
+
+Therefore, using Nx it can be fairly straightforward to scaffold and build a Micro Frontend Architecture from a monorepo with all the additional benefits of Nx.
+
+In this article, I'll show you to how setup a Micro Frontend Architecture with Nx and Angular.
+
+## What we'll build
+
+We will put together a simple Admin Dashboard application that requires a user to log in to view the protected content.  
+To achieve this we will have two apps:
+
+- Admin Dashboard app
+  - This will contain the layout of the dashboard and the content that can only be viewed by an authenticated and authorized user.
+- Login app
+  - This will contain the forms and logic for authenticating and authorizing a user
+
+When the user tries to view a protected page within the Admin Dashboard, if they are not authenticated we will present them with a login form so that they can authenticate and view the contents of the page.
+
+The end result should look something like this:
+
+## First steps
+
+### Create an Nx Workspace
```

Comment:
idk if we need this.. I assume most people that are reading this guide will already have an nx workspace.

Reply:
I think we should keep this. 

It means everyone looking at the guide is at the same starting position and knows how we got there.

I know that personally I hate guides and tutorials that make assumptions